site/styles/_buttons.scss

```diff
@@ -108,73 +106,70 @@
     font-size: 18px;
     font-weight: $font-weight-bold;
   }
+}
 
-  &.s72-btn-trailer {
+.s72-btn-play.s72-btn-trailer {
+  background: transparent;
+  border: 0 none;
+  @include hover-focus {
     background: transparent;
-    border: 0 none;
-    @include hover-focus {
-      background: transparent;
 
-      .s72-icon-play {
-        circle {
-          fill: $primary;
-          stroke: $primary;
-          stroke-width: 3px;
-        }
-
-        path {
-          fill: #000;
-          stroke: #000;
-        }
-      }
+    .s72-icon-play circle {
+      fill: $primary;
+      stroke: $primary;
+      stroke-width: 3px;
+    }
```

Comment:
This is fine I guess, but was there a rationale against nesting? Because nesting seems more idomatic to me...

Reply:
The rationale against **deep** nesting (_I'm not against sensible nesting_) is that it is less readable and less searchable, especially in long classes that require scrolling to read... some people view that as a [smell](https://csswizardry.com/2017/02/code-smells-in-css-revisited/#bem-mixes).  I think the 'BEM mixes' solution in that article is a good idea and we already do that a little bit in some places so perhaps we can gradually push in that direction more.

Reply:
Sure, but the refactored Sass is what I expect the original Sass to compile to in CSS anyway so we diverge from what I consider idiomatic Sass syntax and use more characters for the same result.

Reply:
Note, I'm not attempting to defend the existing deep nesting.

Reply:
Isn't it fewer characters?  And fewer lines too?  :p

I did this refactor and a few others like it before I realised I could add those `stylelint-disable` comments... I can change it back and add the comment if you want but I reckon this way is 1% more human readable even if it's not the Sass way.

Reply:
More chars from having to duplicate `.s72-icon-play`. I don't see the advantage (or negative) of:

```
.s72-icon-play circle { }
.s72-icon-play path { }
```

... over:

```
.s72-icon-play {
  circle { }
  path { }
}
```

This is more words than needed over something so minor, leave it as is 👍